uefi/src/boot.rs

```diff
@@ -44,6 +44,12 @@ use uefi_raw::table::boot::InterfaceType;
 #[cfg(feature = "alloc")]
 use {alloc::vec::Vec, uefi::ResultExt};
 
+/// Size in bytes of a UEFI page.
```

Comment:
Shouldn't we move this to uefi-raw or reexport it from there?

Reply:
Good call, opened https://github.com/rust-osdev/uefi-rs/pull/1383